tts_cli/tts_utils.py

```diff
@@ -99,8 +99,10 @@ def tts(self, text: str, voice: str, outputName: str, output_subfolder: str, for
 
         voice_id = self.voice_map[voice]
         url = f"https://api.elevenlabs.io/v1/text-to-speech/{voice_id}"
+        # use multilingual model to support more languages 
         payload = {
             "text": text,
+            "model_id": "eleven_multilingual_v2",
```

Comment:
there should be an option to change this since it heavily effects the sound of the voices.